dependency/vault_common.go

```diff
@@ -20,7 +22,7 @@ type Secret struct {
 // leaseDurationOrDefault returns a value or the default lease duration.
 func leaseDurationOrDefault(d int) int {
 	if d == 0 {
-		return VaultDefaultLeaseDuration
+		return int(VaultDefaultLeaseDuration.Nanoseconds() / 1000000000)
 	}
```

Comment:
@jefferai `.Seconds()` returns a float64 with decimals for anything less than a second. I could have divided by time.second though